app/api/event_copyright.py

```diff
@@ -40,22 +40,33 @@ class EventCopyrightList(ResourceList):
 
     def query(self, view_kwargs):
```

Comment:
Looks like both were made at the exact same time :stuck_out_tongue: . I'll check that. For others I will make a separate PR.